### DISCUSSION TITLE
Body scroll hidden when opening a dropdown (or any sort of similar thing)

### DISCUSSION THREAD

**Discussion by aervxa** [melt-ui/melt-ui#1281] [Ideas]
Body scroll being hidden when a dropdown is being opened is not always nice, what if it is a navbar dropdown and the user wants to scroll in the page to find where to navigate. Is it even intended, or a workaround?
ps. didn't want to create a github issue lol

**Comment by mcmxcdev** [comment]
You can set the `preventScroll` prop to `false` on a `Select` component to be able to scroll while it is opened.

**Reply by aervxa** [reply]
Oh shit, damn thanks! (im new to melt)
